ui/pages/Login/components/Login.jsx

```diff
@@ -12,6 +16,48 @@ const FIELDS = [
   { name: 'password', label: 'Password', type: 'password', validate: validators.required },
 ]
 
+const googleLogin = () => {
```

Comment:
this should be in the reducers file

Reply:
Okay. Move it to `ui/pages/Login/reducers.js`.